.github/workflows/ci-build-tests.yml

```diff
@@ -37,6 +37,7 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
```

Comment:
Is this change intended to be merged? Will PRs commonly include this line, while in testing, going forward?

Reply:
> Is this change intended to be merged? Will PRs commonly include this line, while in testing, going forward?

No, that's why it's part of the "`**** REVERT ME **** fail-fast: false **** REVERT ME ****`" commit. It's only enabled temporarily so I could easily see which tests were failing without CI stopping early.